src/itype.h

```diff
@@ -14,6 +14,7 @@
 #include "damage.h"
 #include "translations.h"
 #include "calendar.h"
+#include "mongroup.h"
```

Comment:
This is not needed. A forward declaration of `MonsterGroup` and the typedef of `mongroup_id` is enough (see "map.h" for how it's done).